src/lib/ChatWindow.vue

```diff
@@ -562,6 +563,13 @@ export default {
     roomInfo() {
       this.$emit('room-info', this.room)
     },
+    onAvatarClick(id) {
+      if (this.room?.isIndividual) {
+        this.roomInfo()
+      } else {
+        this.$emit('room-info', { senderId: id })
+      }
+    },
```

Comment:
Nesse trecho eu usaria o `return` pra tirar a necessidade do else. É uma prática que uso pra deixar o código mais sucinto.
```suggestion
    onAvatarClick(id) {
      if (this.room?.isIndividual) {
        this.roomInfo()
        return
      }
      this.$emit('room-info', { senderId: id })
    },
```